### PR TITLE
Fixes #174 enable more than one observer

### LINKED ISSUE #174: Enable more than one observed data set in Qualification Plan-ComparisonTimeProfilePlots
@AndreDlm commented on [Wed Feb 26 2020](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/60)

In the Qualification Plan for DDI, section `ComparisonTimeProfilePlots` I'd like to have the option to include more than just one observed data set, e.g. something like:

Annotation 2020-02-26 160007

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: examples/advanced_01/input/master_quali_plan.json [hunk 1]
```diff
@@ -318,7 +318,7 @@
                 "Project": "Midazolam",
                 "Simulation": "Mida iv 0.05 mgkg - Gorski 1998 - Whole blood",
                 "Output": "PeripheralVenousBlood|Whole Blood|Midazolam",
-                "ObservedData": "Gorski et al. 1998, iv 0.05 mgkg, Whole blood, n=16",
+                "ObservedData": ["Gorski et al. 1998, iv 0.05 mgkg, Whole blood, n=16"],
```
Comment (msevestre) [inline examples/advanced_01/input/master_quali_plan.json:321]: @Yuri05 This works too.

File: schemas/OSP_Qualification_Plan_Schema.json [hunk 2]
```diff
@@ -27,6 +27,19 @@
         }
       ]
     },
+    "observedData": {
```
Comment (msevestre) [inline schemas/OSP_Qualification_Plan_Schema.json:30]: @Yuri05 I think this is the way to define multiple values correct?

File: schemas/OSP_Qualification_Plan_Schema.json [hunk 3]
```diff
@@ -772,7 +785,7 @@
                       "$ref": "#/definitions/nonEmptyString"
                     },
                     "ObservedData": {
-                      "$ref": "#/definitions/nonEmptyString"
+                      "$ref": "#/definitions/observedData"
```
Comment (msevestre) [inline schemas/OSP_Qualification_Plan_Schema.json:788]: I am not sure about DDIRatio and PKRatio plot as there is also the ObservedDataRecordId and it seems to be a 1-1 relationship

File: schemas/OSP_Qualification_Plan_Schema.json (hunk 2 above)
Comment (Yuri05) [reply schemas/OSP_Qualification_Plan_Schema.json:30]: I think so.
We could also add `"minItems": 1,` to the array if we always want to have at least 1 obs data

File: schemas/OSP_Qualification_Plan_Schema.json (hunk 3 above)
Comment (Yuri05) [reply schemas/OSP_Qualification_Plan_Schema.json:788]: Yes, I think it's always 1:1 indeed

File: schemas/OSP_Qualification_Plan_Schema.json (hunk 2 above)
Comment (msevestre) [reply schemas/OSP_Qualification_Plan_Schema.json:30]: done

**Pull request merged**